### ISSUE TITLE
Input / text area styling issues

### ISSUE THREAD

**Issue by frasermclean** [starwind-ui/starwind-ui#139]
After updating my Input from 1.3.0 to 1.3.2, it seems that the error and success styles no longer get applied. 

Old relevant styles
```
user-valid:border-success user-invalid:border-error
```

New styles
```
aria-invalid:border-error aria-invalid:focus-visible:ring-error/40
```

It seems like the aria-invalid selector doesn't work. Also we lost the success state too.

As an aside, why is the default background in light mode transparent? There is already bg-input which I would have thought was the appropriate class to use.

**Comment by frasermclean** [comment]
After some more digging, it seems its up to me to programmatically set the aria-invalid state on the input. Is this correct?

**Comment by Boston343** [comment]
That's correct - it was changed to match the shadcn implementation for all form components. You'll find other starwind form components use the same pattern as well.

This also goes for the bg-transparent for light mode - it was updated to match shadcn design. My goal has been to make them very close in design, except using base font size as standard instead of the small size that shadcn uses. So starwind components are slightly larger by default.

**Comment by Boston343** [comment]
Closing as a non-issue
